src/main/js/App.js

```diff
@@ -542,9 +542,19 @@ const App = () => {
                     </div>
                 </div>
                 <div>
-                    <div>{"Console"}</div>
-                    <textarea readOnly value={consoleText} class="console"></textarea>
-                </div>
+                    {process.env.NODE_ENV !== 'production' &&
```

Comment:
Good use of &&. Once we merge this, we should make sure to check that this does NOT appear on heroku.